frontend/www/js/omegaup/components/badge/Badge.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <figure class="badge-container"
           v-tooltip="description">
-    <img class="badge-icon"
-            v-bind:src="this.iconUrl">
+    <a class="badge-icon"
+            v-bind:href="`/badge/${this.badge.badge_alias}`"><img v-bind:src="this.iconUrl"></a>
```

Comment:
```suggestion
            v-bind:href="`/badge/${this.badge.badge_alias}/`"><img v-bind:src="this.iconUrl"></a>
```

tratamos de terminar todos los URLs con `/`.